pkg/api/customization/multiclusterapp/multi_cluster_app.go

```diff
@@ -111,7 +112,7 @@ func (w Wrapper) addProjects(request *types.APIContext) error {
 }
 
 func (w Wrapper) removeProjects(request *types.APIContext) error {
-	mcapp, _, inputProjects, _, err := w.modifyProjects(request, removeProjectsAction)
+	mcapp, _, inputProjects, _, _, err := w.modifyProjects(request, removeProjectsAction)
```

Comment:
Do we have to update the TargetToRole for the final targets after removing projects?